# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../freetype/libtool-link-lib64.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -Naur freetype-2.1.7.orig/builds/unix/ltmain.sh freetype-2.1.7/builds/unix/ltmain.sh
--- freetype-2.1.7.orig/builds/unix/ltmain.sh	2003-11-09 10:37:13.000000000 +0200
+++ freetype-2.1.7/builds/unix/ltmain.sh	2005-03-05 22:56:07.969688072 +0200
@@ -3264,7 +3264,7 @@
 	      fi
 	      if test -n "$a_deplib" ; then
 		libname=`eval \\$echo \"$libname_spec\"`
-		for i in $lib_search_path $sys_lib_search_path $shlib_search_path; do
+		for i in $lib_search_path $sys_lib_search_path $shlib_search_path /lib64 /usr/lib64; do
 		  potential_libs=`ls $i/$libname[.-]* 2>/dev/null`
 		  for potent_lib in $potential_libs; do
 		      # Follow soft links.
